### PR TITLE
Remove note about filtering on FeatureTypes

### PR DESCRIPTION
**Pull request by JensTrier**
Remove note about filtering on FeatureTypes, since it is now available in Get-CsOnlineUser.

### CODE REVIEW

**Comment by JensTrier** [comment]
Adding ProxySbc parameter description to New/Set-CsOnlinePstnGateway

**Comment by JensTrier** [comment]
@yogkumgit

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 1]
```diff
@@ -40,7 +40,6 @@ The following table lists the parameters to `Get-CsOnlineVoiceUser` and the alte
 | NumberNotAssigned | Find users without a phone number assigned | `Get-CsOnlineUser -Filter {(LineUri -eq $Null) -and (FeatureTypes -contains 'PhoneSystem') -and (AccountEnabled -eq $True)} -AccountType User` |
 | PSTNConnectivity | Find enabled users with PhoneSystem (OnPremises) or CallingPlan (Online) | Online: `Get-CsOnlineUser -Filter {(FeatureTypes -contains 'CallingPlan') -and (AccountEnabled -eq $True)} -AccountType User` OnPremises: `Get-CsOnlineUser -Filter {-not (FeatureTypes -contains 'CallingPlan') -and (FeatureTypes -contains 'PhoneSystem') -and (AccountEnabled -eq $True)} -AccountType User` |
 
-**Note**: `Get-CsOnlineUser` filtering on FeatureTypes will be available later this year.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:43]:
```suggestion
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 2]
```diff
@@ -57,7 +56,6 @@ The following table lists the output fields from `Get-CsOnlineVoiceUser` and the
 | Number | LineUri in the output from `Get-CsOnlineUser`. You can get same phone number format by doing LineUri.Replace('tel:+','')  |
 | Location | Use LocationId in the output from `Get-CsPhoneNumberAssignment -AssignedPstnTargetId <Identity>` as the input to `Get-CsOnlineLisLocation -LocationId` |
 
-**Note**: `Get-CsOnlineUser` filtering on FeatureTypes will be available later this year.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:59]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested changes to remove the extra blank lines. Many thanks.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the suggested changes to remove the extra blank lines. Many thanks.

Thanks & Done!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @get-itips @tseward

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**